config/environment/ui-test.php

```diff
@@ -16,8 +16,14 @@
     'tests.ui.url_normalizer_blacklist.api' => [],
     'tests.ui.url_normalizer_blacklist.controller' => [],
 
-    // disable check for plugin updates during UI tests
-    'dev.disable_plugin_update_checks' => true,
+    // disable check for plugin updates during UI tests, allow for override
+    'dev.disable_plugin_update_checks' => Piwik\DI::decorate(function ($previous, Container $c) {
+        if ($c->get('test.vars.forceEnablePluginUpdateChecks')) {
+            return false;
+        }
+
+        return true;
```

Comment:
That could be simplified to:
```suggestion
        return !$c->get('test.vars.forceEnablePluginUpdateChecks');
```